### ISSUE TITLE
Rebrand SanchoNet to Voltaire

### ISSUE THREAD

**Issue by m-i-k-e** [IntersectMBO/govtool#1640]
We need to find a name for GovTool on other networks besides Sancho. We are moving beyond Sancho now.

There is currently no rebrand that's approved, or new name for the app, however I would suggest that we call it "Cardano GovTool". This is in preference to "Voltaire" as Voltaire is not a thing that's being governed, it's a phase of the Cardano blockchain development.

@Ryun1 also tells me that the digital pumpkin is not owned by us, nor do we have the rights to use it. It also delivers very little identity or branding content. It needs to be replaced with something. Possibly a Cardano logo, if we go with "Cardano GovTool" as the name.

@l-br1 Can you please have a look and comment? I'd like to ask the designers to work on this right away. I'm assigning this to you for your attention.

**Comment by l-br1** [comment]
I think we need to re-size what we mean by rebranding.

I would agree that Cardano govtool would be nice, but we don't have the rights to use Cardano, that is why we called it Voltaire Govtool (which is the official name), for short, govtool.

The rebranding in this instance just refers to witching back logos and text (especially on landing page) to refer to Voltaire Govtool instead of sanchonet govtool.

Regarding the pumpkin, it's free to use use, so we can use it, if needed I can pay $49 to get the full set.

So TLDR:
- Name: Voltaire Govtool (logo already exists)
- not the time to start an actual rebranding 
- Action: change logo to the original one and check copy to reference the production tool 

cc: @bosko-m

**Comment by m-i-k-e** [comment]
OK, that's simple enough. @l-br1
